components/Tools/Launcher.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-container justify="space-around">
+    <v-row rows="auto" align-content="center" align="center">
```

Comment:
```suggestion
    <v-row align-content="center" align="center">
```
does row props exist?